ocaml/graph/README.markdown

```diff
@@ -10,24 +9,27 @@ To create a graph showing relationships between
 
 run:
 
-  $ graph -h <host> -u root -pw <password> VM VDI VBD SR > graph.gv
+```bash
+graph.native -h <host> -u root -pw <password> VM VDI VBD SR > graph.dot
+```
 
-install graphviz and then run:
+Then install graphviz and run:
 
-  $ dot -Tpng graph.gv -o graph.png
+```bash
+dot -Tsvg graph.dot -o graph.svg
+```
 
 Here's an example:
 
-![Example rendered graph](../../../../raw/master/ocaml/graph/example.png)
+![Example rendered graph](vm-disks.svg)
```

Comment:
I think this is done on purpose by github for security reasons. There is some info on stackoverflow for this (search for include-an-svg-hosted-on-github-in-markdown).